app/_data/plugins/ai-proxy.yaml

```diff
@@ -0,0 +1,139 @@
+providers:
+  - name: 'OpenAI'
+    formats: 'GPT-3.5, GPT-4, GPT-4o, and Multi-Modal'
+    url_pattern: 'https://api.openai.com:443/{route_type_path}'
+    min_version: '3.6'
+    chat:
+      supported: ✅
+      upstream_path: '/v1/chat/completions'
+      route_type: 'llm/v1/chat'
+      model_example: 'gpt-4'
+    completions:
+      supported: ✅
+      upstream_path: '/v1/completions'
+      route_type: 'llm/v1/completions'
+      model_example: 'gpt-3.5-turbo-instruct'
+    streaming:
+      supported: ✅
+  - name: 'Cohere'
+    url_pattern: 'https://api.cohere.com:443/{route_type_path}'
+    min_version: '3.6'
+    chat:
+      supported: ✅
+      upstream_path: '/v1/chat'
+      route_type: 'llm/v1/chat'
+      model_example: 'command'
+    completions:
+      supported: ✅
+      upstream_path: '/v1/generate'
+      route_type: 'llm/v1/completions'
+      model_example: 'command'
+    streaming:
+      supported: ✅
+  - name: 'Azure'
+    url_pattern: 'https://{azure_instance}.openai.azure.com:443/openai/deployments/{deployment_name}/{route_type_path}'
+    min_version: '3.6'
+    chat:
+      supported: ✅
+      upstream_path: '/openai/deployments/{deployment_name}/chat/completions'
+      route_type: 'llm/v1/chat'
+      model_example: 'gpt-4'
+    completions:
+      supported: ✅
+      upstream_path: '/openai/deployments/{deployment_name}/completions'
+      route_type: 'llm/v1/completions'
+      model_example: 'gpt-3.5-turbo-instruct'
+    streaming:
+      supported: ✅
+  - name: 'Anthropic'
+    url_pattern: 'https://api.anthropic.com:443/{route_type_path}'
+    min_version: '3.6'
+    chat:
+      supported: ✅
+      upstream_path: '/v1/complete in version 3.6, /v1/messages since version 3.7'
+      route_type: 'llm/v1/chat'
+      model_example: 'claude-2.1'
+    completions:
+      supported: ✅
+      upstream_path: '/v1/complete'
+      route_type: 'llm/v1/completions'
+      model_example: 'claude-2.1'
+    streaming:
+      supported: Chat type only
+  - name: 'Mistral'
+    formats: 'mistral.ai, OpenAI, raw, and OLLAMA formats'
+    url_pattern: 'As defined in `{{ upstream_url }}'
```

Comment:
Should this look like this 
![image](https://github.com/user-attachments/assets/8e03be92-976d-42ee-8905-ae911989dbf9)

Reply:
I fixed it, the way I did it is not ideal but we're going to revisit the whole thing later anyway, so I think it's good enough for now